test/util/client.go

```diff
@@ -43,6 +43,9 @@ func GetClusterAdminClient(adminKubeConfigFile string) (*client.Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	// no rate limit
+	clientConfig.QPS = -1
```

Comment:
Do any tests exercise QPS? If so maybe this should be configured in the tests which want to disable it?